projects/admin/src/app/record/detail-view/item-detail-view/item-transaction/item-transaction.component.ts

```diff
@@ -113,6 +113,15 @@ export class ItemTransactionComponent implements OnInit, OnDestroy {
     private _dialogService: DialogService
   ) {}
 
+  /**
+   * Get current pickup location
+   * @return pickup location name
+   */
+  getCurrentPickupLocation(): string {
```

Comment:
It could be done with a getter instead of a method.

Reply:
Done